app/assets/javascripts/shopify_app/top_level_interaction.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  function setUpTopLevelInteraction() {
+    var TopLevelInteraction = new ITPHelper({
+      content: '#TopLevelInteractionContent',
+      action: '#TopLevelInteractionButton',
```

Comment:
I wouldn't consider this code resusable so I wouldn't bother making these selectors configurable, just make them constants in the ITPHelper klass.